mariadb-plugin/pom.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright © 2020 CDAP
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>database-plugins</artifactId>
+        <groupId>io.cdap.plugin</groupId>
+        <version>1.4.0-SNAPSHOT</version>
+    </parent>
+
+    <name>Maria DB plugin</name>
+    <artifactId>mariadb-plugin</artifactId>
+    <version>1.4.0-SNAPSHOT</version>
+    <modelVersion>4.0.0</modelVersion>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.cdap.cdap</groupId>
+            <artifactId>cdap-etl-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.cdap.plugin</groupId>
+            <artifactId>database-commons</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.cdap.plugin</groupId>
+            <artifactId>hydrator-common</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
+
+        <!-- test dependencies -->
+        <dependency>
+            <groupId>io.cdap.plugin</groupId>
+            <artifactId>database-commons</artifactId>
+            <version>${project.version}</version>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.cdap.cdap</groupId>
+            <artifactId>hydrator-test</artifactId>
```

Comment:
scope should be test
https://github.com/data-integrations/google-cloud/blob/develop/pom.xml#L416
Please take a look at above pom file for scope of the artifacts.

Reply:
This dependency has been already declared with scope as "provided" in the parent pom.xml file. Do we still have to explicitly mention the scope here? We checked pom.xml files from other sub-projects under database-plugins and none of them has scope explicitly mentioned. Can you please confirm?